autoscale/src/main/java/com/sequenceiq/periscope/controller/AutoScaleClusterCommonService.java

```diff
@@ -103,7 +108,9 @@ public Cluster setState(Long clusterId, StateJson stateJson) {
 
     public Cluster setAutoscaleState(Long clusterId, AutoscaleClusterState autoscaleState) {
         Cluster cluster = setAutoscaleState(clusterId, autoscaleState.isEnableAutoscaling());
-        return setStopStartScalingState(cluster.getId(), autoscaleState.getUseStopStartMechanism());
+        Boolean enableStopStartScaling = shouldEnableStopStartBasedOnEntitlement(cluster,
+                cluster.getTimeAlerts().size(), autoscaleState.getUseStopStartMechanism());
```

Comment:
This method invocation should also change if the method is refactored

Reply:
Done